pkg/rpc/server/server.go

```diff
@@ -906,13 +906,13 @@ func (s *Server) getUnclaimedGas(ps request.Params) (interface{}, *response.Erro
 	}, nil
 }
 
-// getValidators returns the current NEO consensus nodes information and voting status.
-func (s *Server) getValidators(_ request.Params) (interface{}, *response.Error) {
+// getNextBlockValidators returns validators for the next block with voting status.
+func (s *Server) getNextBlockValidators(_ request.Params) (interface{}, *response.Error) {
 	var validators keys.PublicKeys
 
-	validators, err := s.chain.GetValidators()
```

Comment:
We need to look at other users of `GetValidators` then, maybe it's no longer relevant there either.

Reply:
DBFT still uses this method, we can't remove it.

Reply:
I don't think it needs to. It wants next block validators too.